tools/internal_ci/linux/grpc_e2e_performance_v2.sh

```diff
@@ -77,6 +77,10 @@ buildConfigs() {
         -s big_query_table="${table}" -s timeout_seconds=900 \
         -s prebuilt_image_prefix="${PREBUILT_IMAGE_PREFIX}" \
         -s prebuilt_image_tag="${UNIQUE_IDENTIFIER}" \
+        -a ci_buildNumber="${KOKORO_BUILD_NUMBER}" \
+        -a ci_buildUrl= "https://source.cloud.google.com/results/invocations/" + "${KOKORO_BUILD_NUMBER}" \
+        -a ci_jobName="${KOKORO_JOB_NAME}" \
+        -a ci_gitCommit="${KOKORO_GIT_COMMIT}" \
```

Comment:
what about gitActualCommit?

Reply:
The original mechanism doesn't work since the CI has been moved to Kokoro, "$ghprbActualCommit" was populated by [GitHub Pull Request Builder](https://plugins.jenkins.io/ghprb/), I tried to print this ENV out in Kokoro machine, it is empty there, even we add the annotation there, the field will be empty. 

I left the this field inside the algorithm since we might be able to populate this field through annotation in the future.

I can add the annotation back if that makes more scene.

Reply:
Should I add gitActualCommit back?